client/client.go

```diff
@@ -50,7 +50,7 @@ type ChatGPT struct {
 func NewChatGPT(token string) *ChatGPT {
 	return &ChatGPT{
 		Token: token,
-		Model: openai.Models["gpt-4.1"],
+		Model: openai.Models["gpt-5"],
```

Comment:
Maybe this could be configuration instead of code? After all, it seems like something that would update pretty often.